app/__init__.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+
+import os
+from config import app_config
+#import the configurations
+
+def create_app(config_name):#creates the app with the desired environments, function to create the app instance
+    app = Flask(__name__, instance_relative_config=True)#instantiate the flask app
+    app.url_map.strict_slashes = False#this will set the strict_slashes flag to False for every route created 
+    app.config.from_object(app_config[config_name])
```

Comment:
Put all comments on a new line (Each comment should be on their own lines, not the same line as the code)